homeassistant/const.py

```diff
@@ -171,6 +171,8 @@
 EVENT_LOGBOOK_ENTRY = 'logbook_entry'
 EVENT_THEMES_UPDATED = 'themes_updated'
 EVENT_TIMER_OUT_OF_SYNC = 'timer_out_of_sync'
+EVENT_AUTOMATION_TRIGGER = 'automation_trigger'
```

Comment:
All events are fired after they happen, so we should make this `automation_triggered`